packages/election/graphql/resolvers/_mutations/createElection.js

```diff
@@ -0,0 +1,40 @@
+const moment = require('moment')
+
+const {
+  Roles,
+  ensureSignedIn
+} = require('@orbiting/backend-modules-auth')
+
+const { upsert } = require('../../../lib/db')
+
+module.exports = async (_, { electionInput }, { pgdb, user: me, req }) => {
+  ensureSignedIn(req)
+  Roles.ensureUserIsInRoles(me, ['admin', 'supporter'])
+
+  const {
+    slug,
+    description,
+    beginDate
+  } = electionInput
+
+  const start = moment(beginDate)
```

Comment:
`start` only used once, in template string a little further below; would thus go for:

```
documentPath: `${moment(beginDate).format('/YYYY/MM/DD')}/${slug}`
```

Looks to me like it does not decrease readability.